static/src/stylesheets/module/site-messages/_engagement-banner-us-eoy.scss

```diff
@@ -0,0 +1,235 @@
+@font-face {
+    font-family: 'Guardian Titlepiece';
+    src:
+    url('https://interactive.guim.co.uk/fonts/garnett/GTGuardianTitlepiece-Bold.woff2') format('woff2'),
```

Comment:
Can this not be pulled in via an NPM in order to not proliferate the URL to the font?

Reply:
this is how it's been done in the past, as we are the only ones that use this on frontend - you can't do an include in sass sheet unless it's declared somewhere else (like this)

Reply:
the upside is that when we delete this stylesheet after the campaign, this font will no longer be getting downloaded when someone visits www